molgenis-core-ui/src/main/javascript/modules/react-components/wrapper/Ace.js

```diff
@@ -42,9 +42,11 @@
 		componentDidMount: function() {
 			var container = this.refs.editor.getDOMNode();
 			var editor = ace.edit(container);
+			require('brace/theme/'+this.props.theme);
```

Comment:
Webpack allows for asynchronous imports. This is an example of such an import.
We only retrieve the theme when the component is mounted, to reduce the initial load to the server. Ace brings a lot of kb's to the client, we try to lessen it by asynch requires